### PR TITLE
adds delete by name processor

### PR DESCRIPTION
**Pull request by azharkhan**
- helps in purging specfic messages

### CODE REVIEW

File: test/deadLetterProcessors.js [hunk 1]
```diff
@@ -45,6 +45,27 @@ describe('Deadletter processors', function() {
     });
   });
 
+  // eslint-disable-next-line
```
Comment (earobinson) [inline test/deadLetterProcessors.js]: remove me

**Review by earobinson — APPROVED** [review]
tests!

**Comment by misshula** [comment]
👍

**Comment by earobinson** [comment]
DAVE

**Pull request merged**